### PR TITLE
handle error code from search results

### PR DESCRIPTION
**Pull request by adammeyer**

### CODE REVIEW

File: src/app/searchResults/searchResults.component.js [hunk 1]
```diff
@@ -45,6 +45,9 @@ class SearchResultsController {
           this.searchResults = results;
         }
         this.loadingResults = false;
+      }, () => {
+        this.searchResults = [];
```
Comment (OzzieOrca) [inline src/app/searchResults/searchResults.component.js]: Should we show the user an error message that maybe includes a message to try again?

Also would it be useful to log this to rollbar?

**Comment by OzzieOrca** [comment]
Is this https://jira.cru.org/browse/EP-1595? If so can I assign it to you?

File: src/app/searchResults/searchResults.component.js (hunk 1 above)
Comment (adammeyer) [inline src/app/searchResults/searchResults.component.js]: Yes and yes.  I'll take care of that.

**Comment by adammeyer** [comment]
@OzzieOrca Error message added

**Comment by OzzieOrca** [comment]
Did we want to add a $log statement too?

**Comment by adammeyer** [comment]
I can, do you recommend I do?  I was thinking that's a gateway error, not FE.

**Comment by OzzieOrca** [comment]
Idk. I've been logging most API failures. Does the gateway log every error status code? Logging in give-web would let you know where that failed request originated from. (Although until we get sourcemaps working it's hard to tell exactly)

**Comment by adammeyer** [comment]
@OzzieOrca Let me know if you have any other comments.

**Pull request merged**